### DISCUSSION TITLE
CtrlV.link image upload integration ready

### DISCUSSION THREAD

**Discussion by lukyer** [Chatterino/chatterino2#3513] [Q&A] [answered]
Hi,
this image uploader integration was requested by twitch.tv users. We made it available for you software as follows:
![image](https://user-images.githubusercontent.com/3464342/149637024-d13bfce8-6f38-4d4f-bb33-608e46ef5ea8.png)

Please update your wiki https://wiki.chatterino.com/Image%20Uploader/ manually or show me to where should I add it via PR. I didn't find wiki source code in this repo.

Thanks and keep doing the great work!

**Comment by zneix** [comment, marked as answer]
Hey @lukyer, our documentation is hosted on a [separate repository](https://github.com/chatterino/wiki), feel free to submit a patch there by adding a new table with description above it in https://github.com/Chatterino/wiki/blob/master/docs/Image%20Uploader.md

**Reply by lukyer** [reply]
Thanks, I've crafted PR https://github.com/Chatterino/wiki/pull/164

**Comment by Mm2PL** [comment]
This image uploader setting exhibits the same bug as one of the other popular ones: if you mouse over the deletion link your image will get deleted because the service accepts GET requests on the deletion URL. At least I think, Im tired enough to typo edit.php and I'm not so sure anymore.

**Reply by lukyer** [reply]
Ah thanks, good point, to be sure I added redirect (and updated delete url).

**Comment by Fildous1** [comment]
It would be very helpful if you added information about integrating ctrlv on the wiki, me and many others would use it. I'm asking as a happy user of Chatterino and Ctrlv
